d2bs/kolbot/tools/PickThread.js

```diff
@@ -49,11 +49,14 @@ function main() {
 				break;
 			}
 		}
-		if (!me.inTown && !noPick) {
-			Pickit.fastPick();
+
+		if (!me.inTown && !noPick && !me.itemoncursor && Pickit.gidList.length > 0) {

```

Comment:
is gidList guaranteed to be non null?

Reply:
I don't know what would cause `gidList` to be null. the initialization of the var is `gidList: []` and all that's done to it is push() and shift().